zerver/openapi/zulip.yaml

```diff
@@ -10633,6 +10656,34 @@ paths:
                           ],
                         "result": "success",
                       }
+    patch:
+      operationId: reorder-linkifiers
+      summary: Reorder all linkifiers
+      tags: ["server_and_organizations"]
+      description: |
+        Change the order that [linkifiers](/help/add-a-custom-linkifier)'s
+        regular expression patterns are matched in messages and topics.
+        Useful for defining linkifiers with overlapping patterns.
+
+        **Changes**: New in Zulip 8.0 (feature level 201). Before that,
+        linkifiers are not consistently ordered.
```

Comment:
I think that before that, linkifiers would be processed in order by ID, right. So I'd say "Before that, linkifiers were always processed in order by ID, which meant users would need to delete and recreate them in order to modify the processing order."